src/main/java/racingcar/domain/RandomMovingStrategy.java

```diff
@@ -0,0 +1,25 @@
+package racingcar.domain;
+
+import java.util.Random;
+
+public class RandomMovingStrategy implements MovingStrategy {
+
+    private static final int MOVE_CONDITION = 4;
+    private final Random random;
+    public static final int MAX_BOUND = 10;
```

Comment:
`상수` - `인스턴스 변수` 순으로 작성해주시면 좋을 것 같습니다. 구분을 위ㅐ 공백 라인까지 추가해봐도 좋구요.
```suggestion
    private static final int MOVE_CONDITION = 4;
    private static final int MAX_BOUND = 10;
    
    private final Random random;
```